File: c/udemy/4-condicional/lanchonete.c
// Uma lanchonete possui vários produtos. Cada produto possui um código e um preço. Você deve fazer um programa para ler o código e a quantidade comprada de um produto (suponha um código válido), e daí informar qual o valor a ser pago, com duas casas decimais, conforme tabela de produtos ao lado

#include <stdio.h>

int main(){

    int cod, qtd;
    double valor;

    printf("Codigo do produto: ");
    scanf("%d", &cod);
    printf("Quantidade comprada: ");
    scanf("%d", &qtd);

    if (cod == 1){
        valor = 5.00 * qtd;
    }
    else if (cod == 2){
        valor = 3.50 * qtd;
    }
    else if (cod == 3){
        valor = 4.80 * qtd;
    }
    else if (cod == 4){
        valor = 8.90 * qtd;
    }
    else if (cod == 5){
        valor = 7.32 * qtd;
    }

    printf("Valor a pagar: %.2lf\n", valor);

    return 0;
}

File: python/udemy/4-estrutura-condicional/lanchonete.py
# (adaptado de URI 1038) Uma lanchonete possui vários produtos. Cada produto possui um código e um preço. Você deve fazer um programa para ler o código e a quantidade comprada de um produto (suponha um código válido), e daí informar qual o valor a ser pago, com duas casas decimais, conforme tabela de produtos apresentada.
# Usar apenas estruturas condicionais, sem listas ou dicionários.

cod = int(input('Código do produto: '))
quant = int(input('Quantidade: '))

if cod == 1:
    valor = quant * 5
elif cod == 2:
    valor = quant * 3.5
elif cod == 3:
    valor = quant * 4.8
elif cod == 4:
    valor = quant * 8.9
elif cod == 5:
    valor = quant * 7.32
else:
    print('Código inválido!')

print(f'Valor a ser pago: R${valor:.2f}')